README.md

```diff
@@ -69,11 +67,9 @@ pip install flashinfer -i https://flashinfer.ai/whl/cu121/torch2.4/
 
 #### TensorRT-LLM
 
-Ensure [OpenMIP](https://www.open-mpi.org/software/ompi/v5.0/) is installed
-
```

Comment:
I ignored this line when I tested and jumped straight to local installation. I still reproduced the results.